pages/cart.tsx

```diff
@@ -82,50 +85,51 @@ const CartView : NextPage = () => {
         </Title>
         {isLoading ? <CircularProgress />
           : numberOfItems > 0
-        && (
-          <CartItems>
-            {cartContent.map((item) => (
-              <CartItem key={item.id}>
-                <Card elevation="raised">
-                  <Link href={{ pathname: "/assets/[id]", query: { id: item.id } }} title={item.name}>
-                    <CardHeader>
-                      <CardHeaderTitle>
-                        {/* This is just a dummy example */}
-                        {item.domain && item.domain.length > 0 && (
-                          <Tags>
-                            {item.domain.map((domain) => <Chip key={domain} style={{ display: "inline-block" }}>{domain}</Chip>)}
-                          </Tags>
-                        )}
-                        <Typography variant="h2">{item.name}</Typography>
-                      </CardHeaderTitle>
-                    </CardHeader>
-                    <CardContent>
-                      <TruncatedDescription variant="body_long" lines={3} dangerouslySetInnerHTML={{ __html: item.description }} />
-                    </CardContent>
-                  </Link>
-                </Card>
-
-              </CartItem>
-            ))}
-          </CartItems>
-        )}
+          && (
+            <CartItems>
+              {cartContent.map((item) => (
+                <CartItem key={item.id}>
+                  <Card elevation="raised">
+                    <Link href={{ pathname: "/assets/[id]", query: { id: item.id } }} title={item.name}>
+                      <CardHeader>
+                        <CardHeaderTitle>
+                          {/* This is just a dummy example */}
+                          {item.domain && item.domain.length > 0 && (
+                            <Tags>
+                              {item.domain.map((domain) => <Chip key={domain} style={{ display: "inline-block" }}>{domain}</Chip>)}
+                            </Tags>
+                          )}
+                          <Typography variant="h2">{item.name}</Typography>
+                        </CardHeaderTitle>
+                      </CardHeader>
+                      <CardContent>
+                        <TruncatedDescription variant="body_long" lines={3} dangerouslySetInnerHTML={{ __html: item.description }} />
+                      </CardContent>
+                    </Link>
+                  </Card>
 
-        <Banner variant="warning">
-          {intl.formatMessage({ id: "cart.banner.warning" })}
-        </Banner>
+                </CartItem>
+              ))}
+            </CartItems>
+          )}
+        <BannerContainer>
```

Comment:
As discussed, it's better to remove this container and add the space to the banner component itself